<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="QA-HIMANSHU-RANJAN" failures="3" tests="9" name="UAT_SIP2.Cancelled_Booking_Sorting" time="16.585" errors="0" timestamp="2025-04-01T17:05:23 IST" skipped="0">
  <testcase classname="UAT_SIP2.Cancelled_Booking_Sorting" name="Click_On_Cancelled_Booking" time="2.102"/>
  <system-out/>
  <testcase classname="UAT_SIP2.Cancelled_Booking_Sorting" name="Click_On_Items_Per_Page" time="0.088"/>
  <system-out/>
  <testcase classname="UAT_SIP2.Cancelled_Booking_Sorting" name="Sale_Executive_Name_Sorting" time="2.229">
    <failure message="(Cancelled-Booking) Total Count of Sales Executive Name In Lowercase: 4
The following Sales Executive Name are not in uppercase: Manish Sharma, Rakesh Kumawat, Rakesh Kumawat, Zaid Qureshi" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: (Cancelled-Booking) Total Count of Sales Executive Name In Lowercase: 4
The following Sales Executive Name are not in uppercase: Manish Sharma, Rakesh Kumawat, Rakesh Kumawat, Zaid Qureshi
at org.testng.Assert.fail(Assert.java:99)
at UAT_SIP2.Cancelled_Booking_Sorting.Sale_Executive_Name_Sorting(Cancelled_Booking_Sorting.java:79)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Sale_Executive_Name_Sorting -->
  <system-out/>
  <testcase classname="UAT_SIP2.Cancelled_Booking_Sorting" name="Cutomer_Name_Sorting" time="2.192"/>
  <system-out/>
  <testcase classname="UAT_SIP2.Cancelled_Booking_Sorting" name="verify_Booking_Date_Sorting" time="1.315"/>
  <system-out/>
  <testcase classname="UAT_SIP2.Cancelled_Booking_Sorting" name="Model_Name_Sorting" time="2.191">
    <failure message="(Cancelled-Booking) Total Count of Model name In Lowercase: 7
The following Model name are not in uppercase: Dazzling New Dzire, Epic New Swift, Epic New Swift, Epic New Swift, Epic New Swift, Epic New Swift, Epic New Swift" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: (Cancelled-Booking) Total Count of Model name In Lowercase: 7
The following Model name are not in uppercase: Dazzling New Dzire, Epic New Swift, Epic New Swift, Epic New Swift, Epic New Swift, Epic New Swift, Epic New Swift
at org.testng.Assert.fail(Assert.java:99)
at UAT_SIP2.Cancelled_Booking_Sorting.Model_Name_Sorting(Cancelled_Booking_Sorting.java:176)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Model_Name_Sorting -->
  <system-out/>
  <testcase classname="UAT_SIP2.Cancelled_Booking_Sorting" name="Varient_Sorting" time="2.205">
    <failure message="Sorting failed (Cancelled-Booking) in Varient: Lists differ at element [0]: MARUTI ALTO K10 VXI+ (O) 1L 5MT != MARUTI SWIFT ZXI+ expected [MARUTI ALTO K10 VXI+ (O) 1L 5MT] but found [MARUTI SWIFT ZXI+]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Sorting failed (Cancelled-Booking) in Varient: Lists differ at element [0]: MARUTI ALTO K10 VXI+ (O) 1L 5MT != MARUTI SWIFT ZXI+ expected [MARUTI ALTO K10 VXI+ (O) 1L 5MT] but found [MARUTI SWIFT ZXI+]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:1103)
at UAT_SIP2.Cancelled_Booking_Sorting.Varient_Sorting(Cancelled_Booking_Sorting.java:219)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Varient_Sorting -->
  <system-out/>
  <testcase classname="UAT_SIP2.Cancelled_Booking_Sorting" name="Colour_Sorting" time="2.143"/>
  <system-out/>
  <testcase classname="UAT_SIP2.Cancelled_Booking_Sorting" name="Booking_ID_Sorting" time="2.120"/>
  <system-out/>
</testsuite> <!-- UAT_SIP2.Cancelled_Booking_Sorting -->
